### PR TITLE
Fix ODIS url in mainnet bug from rename

### LINKED ISSUE #4764: [Wallet] Failed to retrieve phone number secret error thrown during verification
### Expected Behavior

Able to retrieve phone number secret 

### Current Behavior

On the latest build https://github.com/celo-org/celo-monorepo/releases/tag/wallet-v1.0.0(13) , `Failed to retrieve phone number secret error thrown during verification` is thrown during all verification attempts that I have test locally. 

Logs: 
Info [2020-08-21T22:42:38.790Z] identity/privateHashing@fetchPhoneHashPrivate :: Unknown error :: Unknown failure 404 in Error: Unknown failure 404
    at anonymous (address at index.android.bundle:1:2660947)
    at call 
Info [2020-08-21T22:42:38.793Z] identity/verification :: Error occured during verification flow :: saltFetchFailure in Error: saltFetchFailure
    at anonymous (address at index.android.bundle:1:3146186)
    at call (na
Info [2020-08-21T22:42:38.849Z] ValoraAnalytics/Tracking event error_displayed with properties: {"timestamp":1598049758848,"sessionId":"e6f443668f98f5d0a4f6e79ad4bab0db770448045551fb669e3018cf67716e","userAddress":"0x72311150732b2793d1306405d9a0ff1743642b00","celoNetwork":"mainnet","error":"saltFetchFailure"}

Logs with more details here: https://groups.google.com/a/clabs.co/g/support/c/Y7AyBNh9Gyg

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

Fix ODIS url in mainnet bug from rename

Fixes #4764 

### Backwards compatibility

Yes

### CODE REVIEW

**Review by annakaz — APPROVED** [review]
💯

**Review by jeanregisser — COMMENTED** [review]
🚀

File: packages/mobile/src/geth/networkConfig.ts [hunk 1]
```diff
@@ -101,7 +101,7 @@ const networkConfigs: { [testnet: string]: NetworkConfig } = {
     syncMode: DEFAULT_SYNC_MODE,
     initiallyForno: FORNO_ENABLED_INITIALLY,
     blockchainApiUrl: 'https://blockchain-api-dot-celo-mobile-mainnet.appspot.com/',
-    odisUrl: 'https://us-central1-celo-odis-mainnet.cloudfunctions.net',
+    odisUrl: 'https://us-central1-celo-pgpnp-mainnet.cloudfunctions.net/',
```
Comment (jeanregisser) [inline packages/mobile/src/geth/networkConfig.ts]: Should we open an issue to track hosting this under a new name?

**Comment by annakaz** [comment]
Fixes https://github.com/celo-org/celo-monorepo/issues/4764

File: packages/mobile/src/geth/networkConfig.ts (hunk 1 above)
Comment (jmrossy) [reply packages/mobile/src/geth/networkConfig.ts]: Will do

Comment (jmrossy) [reply packages/mobile/src/geth/networkConfig.ts]: https://github.com/celo-org/celo-monorepo/issues/4773

**Pull request merged**